templates/worker/01-worker-kubelet/alibabacloud/units/kubelet.service.yaml

```diff
@@ -10,6 +10,7 @@ contents: |
   [Service]
   Type=notify
   ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
+  ExecStartPre=-/usr/sbin/restorecon /usr/local/bin/kubenswrapper /usr/bin/kubensenter``
```

Comment:
Belated review but...stray typo here.

(I am still aware of no reason we can't convert the kubelet units to have systemd drop-ins per platform instead of copy-pasta)